build.gradle

```diff
@@ -167,11 +167,11 @@ dependencies {
     implementation "org.hibernate.validator:hibernate-validator-annotation-processor:6.2.3.Final"
 
     // Hibernate
-    implementation 'org.hibernate:hibernate-c3p0:5.4.6.Final'
+    implementation 'org.hibernate:hibernate-c3p0:5.6.7.Final'
```

Comment:
Is the hibernate criteria removal what's stopping us from upgrading to 6.0.0? Is that happening in a separate card?

Reply:
It seemed more prudent to go to the highest version 5 as there was an issue that was fixed in the PR.
The criteria removal will definitely be a larger change as well as my hesitancy to jump on an x.0.0 release.
My opinion is to make a separate card for a v6 jump and give it some time to settle in on the hibernate side.

Reply:
ack